e2etest/declarativeRunner.go

```diff
@@ -35,7 +35,7 @@ import (
 
 var validCredTypesPerLocation = map[common.Location][]common.CredentialType{
 	common.ELocation.Unknown(): {common.ECredentialType.Unknown(), common.ECredentialType.Anonymous(), common.ECredentialType.OAuthToken()}, // Delete!
-	common.ELocation.File():    {common.ECredentialType.Anonymous()},
+	common.ELocation.File():    {common.ECredentialType.Anonymous(), common.ECredentialType.OAuthToken()},
```

Comment:
tests that choose to run on all combinations of cred types will now run all valid OAuth combinations for file, so tests for OAuth are now covered. In addition, OAuth OAuth is supported for BlobFile and FileFile scenarios. 

From testing, it seems that FileBlob is not supported yet.